src/storage.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+const fs = require("fs");
+const fsextra = require("fs-extra");
+const path = require("path");
+const crypto = require("crypto");
+const helper = require("./helper");
+
+class Storage {
+	constructor() {
+		this.references = new Map();
+
+		// Ensures that a directory is empty.
+		// Deletes directory contents if the directory is not empty.
+		// If the directory does not exist, it is created.
+		fsextra.emptyDirSync(helper.getStoragePath());
```

Comment:
> I'm still unsure about clearing at startup. 

Hmm? Why? Currently, history is empty at startup, so it makes sense for nothing to be there. When we add history loading, then maybe it'll be different.

> More importantly, a FS could still be filled until server gets restarted.

Yeah, but I do think this is a good first step, personally. What's your suggestion for this first PR?